### PR TITLE
Do not force host in default configuration (fixes #218)

### LINKED ISSUE #218: Setting kinto.http_host disturbs redirections
Scenario at PyconFr: 

- user A runs kinto on `0.0.0.0:8888` with make serve
- user B tries accesses on user A instance through local Wifi `192.168.1.12:8888`
- user B is redirected to `http://localhost:8888/v1`

Indeed, in default configuration `kinto.http_host` is equal to `localhost:8888`. And we use this setting to force `request.host` if it is defined.

We had done that in cliquet a while ago, because otherwise the pagination  `Next-Page` URL would not contain the port.

I realize this is not acceptable :) Especially because the same information is duplicated several times:

* `[server:main] host = 0.0.0.0 port = 8888`
* `kinto.http_host = 0.0.0.0:8888`
* `[uwsgi] http-socket = 127.0.0.1:8888`

### PR DESCRIPTION
**Pull request by leplatrem**
@Natim r?

### CODE REVIEW

File: config/kinto.ini [hunk 1]
```diff
@@ -75,6 +74,9 @@ multiauth.policies = basicauth
 # kinto.statsd_url = udp://localhost:8125
 # kinto.statsd_prefix = kinto-prod
 
+# kinto.http_scheme = http
+# kinto.http_host = 0.0.0.0:8888
```
Comment (Natim) [inline config/kinto.ini]: This should never look like that, I should be ``kinto.http_host = localhost:8888``

**Comment by Natim** [comment]
Most of what I think is described in #218. Falling back to WSGI might be a good idea on most setup yes.

**Comment by leplatrem** [comment]
Thanks @Natim for the detailed explanation :) Now that you said it I feel dumb :D 

I updated it !

File: config/kinto.ini [hunk 2]
```diff
@@ -83,7 +85,7 @@ multiauth.policies = basicauth
 # [uwsgi]
 # wsgi-file = app.wsgi
 # enable-threads = true
-# http-socket = 127.0.0.1:8888
+# http-socket = /run/wsgi/kinto.sock
```
Comment (Natim) [inline config/kinto.ini]: As far as I know it should be 

```
# [uwsgi]
# wsgi-file = app.wsgi
# enable-threads = true
# socket = /run/uwsgi/kinto-latest.sock
# chmod-socket = 666
```

**Comment by Natim** [comment]
r+ with nit on the uwsgi config names.

**Comment by Natim** [comment]
Super

**Pull request merged**